src/components/implementation/no_interface/llbooter/boot_deps.h

```diff
@@ -94,9 +125,14 @@ boot_newcomp_create(spdid_t spdid, struct cos_compinfo *comp_info)
 	/* Copy sinv into new comp's capability table at a known location (BOOT_SINV_CAP) */
 	cos_cap_cpy_at(new_comp_cap_info[spdid].compinfo, BOOT_SINV_CAP, &boot_info, sinv);
 
+	/* HACK: Currently assume comp 0 is the only interface */
+	if (spdid > 1) {
+		boot_newcomp_sinv_alloc(spdid, 1);
+	}	
```

Comment:
you should use `FIXME`. This doesn't seem like a hack to me. 
Does this mean, this PR is for supporting one interface in booter? isn't generic to multiple interfaces or different component interfaces?